<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketsUI</class>
 <widget class="QWidget" name="TicketsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ticket Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel_2">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="idLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="clientLabel">
        <property name="text">
         <string>Client:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="clientComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="matchLabel">
        <property name="text">
         <string>Match:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="matchComboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="seatLabel">
        <property name="text">
         <string>Seat:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="seatComboBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="priceLabel">
        <property name="text">
         <string>Price:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="priceLineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="statusComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="ticketsTableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addTicketButton">
       <property name="text">
        <string>Add Ticket</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteTicketButton">
       <property name="text">
        <string>Delete Ticket</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateTicketButton">
       <property name="text">
        <string>Update Ticket</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>